docs/operator-manual/policy-servers/02-private-registry.md

```diff
@@ -1,25 +1,35 @@
 ---
-sidebar_label: "Using private registry"
+sidebar_label: "Using private registries"
 title: ""
 ---
 
-# Configuring Policy Servers to use private registry
+# Configuring PolicyServers to use private registries
 
-It is not uncommon that enterprises using cloud native environments have their
-own container registries to store their container images and other artefacts
-used to run their applications. As these artifacts are, most of the time, private,
-it does not make sense to let them be publicly available on the web or even to
-other teams within the same organization. In these scenarios, it is common to
-use private registries that require authentication to access the resources stored
-In them. With this in mind, we have made it possible to specify and configure the credentials
-used to authenticate into a private registry where your policies are stored and
-allowing Kubewarden Policy Server to download policies from public and private
-registries.
+It is possible to configure PolicyServers to use credentials of private OCI
+registries. This will allow those PolicyServers to download policies from
+public and private registries.
 
+Once a PolicyServer is configured to access private registries, policies running
```

Comment:
This sentence is very long. Could we break it down into two separate sentences, please, as below?

Reply:
True! applied the suggestions, thanks!